tfe/data_source_oauth_client_test.go

```diff
@@ -10,6 +10,8 @@ import (
 )
 
 func TestAccTFEOAuthClientDataSource_basic(t *testing.T) {
+	skipIfFreeOnly(t)
```

Comment:
Huh, seems odd that this test would fail for a free org! Free orgs should be able to create/read oauth clients just fine.

Reply:
Ah, this requires a `github_token`, I mis-read the test error. I'll update it